### PR TITLE
Remove Methode related code

### PR DESCRIPTION
**Pull request by MiroslavGatsanoga**
# Description

## What

Remove Methode related code.

## Why

[JIRA ticket](https://financialtimes.atlassian.net/browse/UPPSF-2818).

## Scope and particulars of this PR (Please tick all that apply)

- [X] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [ ] Feature
- [ ] Documentation
- [ ] Breaking change
- [ ] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

### CODE REVIEW

File: healthcheck.go [hunk 1]
```diff
@@ -55,11 +55,6 @@ type readEnvironmentHealthcheck struct {
 
 const pam_run_book_url = "https://runbooks.in.ft.com/publish-availability-monitor"
 
-var readCheckEndpoints = map[string]func(string) (string, error){
-	"S3": buildAwsHealthcheckUrl,
```
Comment (atanasdinov) [inline healthcheck.go:59]: The `buildAwsHealthcheckUrl` along with its test usages could also be removed.

File: checks/preCheck.go [hunk 2]
```diff
@@ -55,88 +44,11 @@ func mainPreCheck(publishedContent content.Content, tid string, publishDate time
...
-	icValRes := publishedContent.Validate(internalComponentsValidationEndpoint, tid, usr, pass)
-	if !icValRes.IsValid {
-		log.Infof("Message [%v] with UUID [%v] has INVALID internal components, skipping internal components schedule check.", tid, publishedContent.GetUUID())
-		return false, nil
-	}
-
-	return true, &SchedulerParam{eomFileForInternalComponentsCheck, publishDate, tid, icValRes.IsMarkedDeleted, metricContainer, environments}
-}
-
 func getValidationEndpointKey(publishedContent content.Content, tid string, uuid string) string {
```
Comment (atanasdinov) [inline checks/preCheck.go:47]: A nitpick but this function is now equivalent to `publishedContent.GetType()` call and is used only in once place so we can remove it altogether.

**Review by atanasdinov — CHANGES_REQUESTED** [review]
Looks great to me! Aside from the minor comments, I also noticed that there are stale examples in the `example-config.json` and the UUID resolving is not used anymore and could be removed.

Happy to approve after that's addressed.

**Review by atanasdinov — APPROVED** [review]

**Review by martin-stanchev — APPROVED** [review]
LGTM

**Pull request merged**